src/pages/code-search.tsx

```diff
@@ -26,32 +26,34 @@ export const CodeSearchPage: FunctionComponent = () => (
                             Onboard to a new codebase, find answers faster, and identify security risks with universal
                             code search.
                         </p>
-                        <div className="pt-1">
+                        <div className="d-flex flex-column flex-lg-row max-w-sm-400 pt-1">
                             <Link
-                                href="/get-started"
+                                href="/demo"
                                 passHref={true}
                                 data-button-style={buttonStyle.primary}
                                 data-button-location={buttonLocation.hero}
                                 data-button-type="cta"
```

Comment:
These data attributes need to be on the `a` element since they won't be passed down and rendered from the Link element. ([note](https://github.com/sourcegraph/about/pull/5339#discussion_r869454427))

Reply:
Ah that's right, thanks! I fixed all the other instances these attributes are on the wrong tag too here [2351b8c](https://github.com/sourcegraph/about/pull/5387/commits/2351b8c24b3fb5bc3205fb2107e89e5144abddb5)